docs/source/allen.rst

```diff
@@ -70,20 +70,21 @@ Single image download
 
 Full-blown registration
 ~~~~~~~~~~~~~~~~~~~~~~~
-The most useful function is the :code:`download_dataset`. It expects the user to specify the unique
-:code:`dataset_id`. It returns a generator. Each iteration then yields
+The most useful function is the :code:`download_dataset_parallel`.
```

Comment:
Can we discuss the difference between
- `` :code:`stuff` ``
- `` `stuff` ``
- ``` ``stuff`` ```

?

According to the [some docs](https://docutils.sourceforge.io/docs/user/rst/quickref.html#inline-markup)
- `` `stuff` `` = interpreted text
- ``` ``stuff`` ``` = inline literal

Usually when I want monospaced text in `rst` I go for ``` ``stuff`` ```. Is `` :code:`stuff` `` going to add some additional functionality/highlighting?

Is `` :code:`stuff` `` part of the `rst` syntax or is it specifi to Sphinx?

Reply:
Hmm. I actually always go for `` :code:`whatever` `` when I want to enforce code highlighting. I did not know that ` ``stuff``  ` has exactly the same effect! 

Assuming they are identical, ` ``stuff`` ` is probably the less verbose option and I am totally fine with using it everywhere. What do you think?

Reply:
Anyway, this PR did not introduce this syntax. I propose we create an issue and that we do not try to solve it in this PR.

Reply:
Good idea. Issue: #61